### ISSUE TITLE
Releases in sync with Eclipse hawkBit 

### ISSUE THREAD

**Issue by volphy** [eclipse/hawkbit-extensions#56]
Would it be possible to release hawkBit Extensions in sync with Eclipse hawkBit?

For example, 0.3.0M6 release of hawkBit Extensions would be compatible with 0.3.0M6 Eclipse hawkBit release.

At the minimum, GitHub releases will be useful.
Ideally, releases in Maven Central would be even better.

**Comment by schabdo** [comment]
In the past we always shyed away from the administrative expense coming along with keeping extensions "released" in sync with the main line. Especially because we encountered not many reasons in the past for adopting extensions due to braking code changes in the main line. However meanwhile things have changed: Eclipse processes improved and the usage of extensions within the community grew a lot in the context of public clouds. So we get more and more feedback from the hawkBit community (like this issue) that a joined tag/release in sync with main line would be of great help.

Having this said I can't promise full availability of Maven Central artefacts (as of today) but GitHub releases would be definitely a good step in this direction. So I created the upcoming extensions [milestone M7](https://github.com/eclipse/hawkbit-extensions/milestone/1) and aligned all PRs which will make it into the next milestone
